subiquity/controllers/snaplist.py

```diff
@@ -36,6 +36,9 @@ class SampleDataSnapInfoLoader:
     def __init__(self, model, snap_data_dir):
         self.model = model
         self.snap_data_dir = snap_data_dir
+        self.snap_list_fetched = False
+        self.all_snap_info_fetched = False
```

Comment:
These two booleans seem to be set in pairs, do we need both?

Reply:
Well, there is one place where they're not set together but nothing actually reads all_snap_info_fetched so that one can indeed go.